src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
```

Comment:
It's important to ensure that the Injector class follows the Singleton design pattern correctly. If it does, `Injector.getInjector()` should return the same instance of Injector every time it's called. Make sure the implementation in `Injector.java` adheres to this pattern.

Reply:
ok, done